adapter/src/main/java/com/twilio/kudu/dataloader/generator/UniformIntegerValueGenerator.java

```diff
@@ -30,9 +30,15 @@ public UniformIntegerValueGenerator(final int minVal, final int maxVal) {
     this.maxValue = maxVal;
   }
 
+  /**
+   * Generates a int value between [minValue, maxValue)
+   */
   @Override
-  public synchronized Integer getColumnValue() {
+  public Integer getColumnValue() {
```

Comment:
Should this be `synchronized` still? Is accessing `random` thread safe? . Is `random` `ThreadLocalRandom`?

Reply:
Good point, modified all the generators to use `ThreadLocalRandom`